### PR TITLE
PREAPPS-6832: Classic UI - Briefcase Share Files

### PR DESCRIPTION
**Pull request by miteshsavani810**

### CODE REVIEW

File: WebRoot/js/zimbraMail/briefcase/model/ZmBriefcaseItem.js [hunk 1]
```diff
@@ -367,6 +367,19 @@ function(node) {
...
+	if (node.acl && node.acl.grant && node.acl.grant.length > 0) {
+		for (var i = 0; i < node.acl.grant.length; i++) {
+			var grant = node.acl.grant[i];
+			this.shares.push(ZmShare.createFromJs(this, grant));
+		}
+	}
+
+	if (node.acl && node.acl.grant) {
+		this.acl = node.acl;
+	}
```
Comment (silentsakky) [inline WebRoot/js/zimbraMail/briefcase/model/ZmBriefcaseItem.js:381]:
```suggestion
	if (node.acl && node.acl.grant) {
		this.acl = node.acl;
		
		if (node.acl.grant.length > 0) {
		    for (var i = 0; i < node.acl.grant.length; i++) {
			    var grant = node.acl.grant[i];
			    this.shares.push(ZmShare.createFromJs(this, grant));
		    }
	        }
	}
```

File: WebRoot/js/zimbraMail/share/model/ZmShare.js [hunk 2]
```diff
@@ -534,6 +534,9 @@ ZmShare.prototype.hasPrivateAccess = function() { return this.isPermAllowed(ZmSh
  */
 ZmShare._getFolderType =
 function(view) {
+	if (view === ZmItem.BRIEFCASE_ITEM) {
+		return 'Briefcase File';
```
Comment (silentsakky) [inline WebRoot/js/zimbraMail/share/model/ZmShare.js:538]: translation?

File: WebRoot/js/zimbraMail/briefcase/model/ZmBriefcaseItem.js (hunk 1 above)
Comment (miteshsavani810) [reply WebRoot/js/zimbraMail/briefcase/model/ZmBriefcaseItem.js]: Done.

File: WebRoot/js/zimbraMail/share/model/ZmShare.js (hunk 2 above)
Comment (miteshsavani810) [reply WebRoot/js/zimbraMail/share/model/ZmShare.js]: Added.

**Review by k-kato — APPROVED** [review]
LGTM

File: WebRoot/js/zimbraMail/core/ZmMimeTable.js [hunk 3]
```diff
@@ -60,6 +60,10 @@ ZmMimeTable.APP_XML     			= "application/xml";
 ZmMimeTable.APP_ZIMBRA_DOC			= "application/x-zimbra-doc";
 ZmMimeTable.APP_ZIP					= "application/zip";
 ZmMimeTable.APP_ZIP2				= "application/x-zip-compressed";
+ZmMimeTable.APP_RTF					= "application/rtf";
+ZmMimeTable.APP_OPENDOC_ODT			= "application/vnd.oasis.opendocument.text";
+ZmMimeTable.APP_OPENDOC_ODP 		= "application/vnd.oasis.opendocument.presentation";
+ZmMimeTable.APP_OPENDOC_ODS 		= "application/vnd.oasis.opendocument.spreadsheet";
```
Comment (k-kato) [inline WebRoot/js/zimbraMail/core/ZmMimeTable.js:66]: just in case, don't we need to add them to `ZmMimeTable._table` around line 118?

Comment (miteshsavani810) [reply WebRoot/js/zimbraMail/core/ZmMimeTable.js:66]: Thanks for pointing out this.
I have added entry of this application types.

**Review by k-kato — APPROVED** [review]

**Pull request merged**